src/views/AddItem.jsx

```diff
@@ -57,32 +57,39 @@ export function AddItem({ listToken, itemList }) {
 	};
 
 	return (
-		<div className="shopping-form-container">
+		<div className=" w-full mx-auto bg-white p-6">
 			{message ? (
-				<div>
-					<h2>{message}</h2>
+				<div className="w-full flex flex-col items-center">
+					<h2 className="text-center text-blue-500 font-semibold text-base drop-shadow-xl">
+						{message}
+					</h2>
 					<Link to="/list">
-						<button>View the list</button>
+						<button className="bg-green-500 py-1 px-3 mt-4 mb-2 rounded text-white hover:text-green-700 hover:bg-transparent hover:border-2 hover:font-normal">
+							View your list
+						</button>
 					</Link>
 				</div>
 			) : (
-				<form onSubmit={handleSubmit}>
+				<form onSubmit={handleSubmit} className="w-full px-4">
 					<FormInput
-						label="Item name"
+						label="Add an item here:"
```

Comment:
It's nice that you added a more user friendly label here and placeholder underneath :slightly_smiling_face: